docs/user_guide/content_types/cleaning_orphans.rst

```diff
@@ -0,0 +1,16 @@
+Handling Orphaned Entities
+==========================
+It is common for site developers to work directly with Chado, especially as they become more familiar with it. And sometimes, there may purpsefully or accidentally remove data that is published in Tripal. This results in an entity being "orphaned".  The entity will still have a page on the website but the data about that entity is no longer available.  If this occurs you can easily remove published entities that are "orphaned" by navigating to **Administer > Tripal Content Types** and clicking the **Unpublish Orphaned Content** link.  The following page appears.
+
+.. image:: cleaning_orphans.1.png
+
+Next, select the content type that has missing data in the **Content Type** select box.  For example, suppose an organism was removed:
+
+.. image:: cleaning_orphans.2.png
+
+A list of at most 10 entities that are orphaned are shown. If the content type has no orphaned entities then nothing is shown.  You can submit a job to clean the orphaned entities by clicking the 
```

Comment:
please clarify: are all orphans deleted or just the ones in the list.

Reply:
Good point. I've updated the code.  Thanks for reading it carefully.